File: lc-files/java/lc412.java
import java.util.ArrayList;
import java.util.List;

public class lc412 {
    public static List<String> fizzBuzzAgain (int target) {
        ArrayList<String> sample = new ArrayList<>();
        int count = 0;

        while(count < target) {
            if(count % 3 == 0) {
                if(count % 5 == 0) {
                    sample.add("FizzBuzz");
                } else {
                    sample.add("Fizz");
                }
            } else if(count % 5 == 0) {
                sample.add("Buzz");
            } else {
                sample.add(String.valueOf(count));
            }
        }

        return sample;
    }
}

File: lc-files/python/lc412.py
def fizzBuzz (num):
    tempArr = [];
    count = 0;

    while count < num:
        if count % 3 == 0:
            if count % 5 == 0:
                tempArr.insert("FizzBuzz");
            else:
                tempArr.insert("Fizz");
        elif(count % 5 == 0):
            tempArr.insert("Buzz");
        else:
            tempArr.push(str(count));

        count = count + 1;
    return tempArr;
